4-HTTP/client/client.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+
+const baseURL = "http://localhost:3000";
+const instance = axios;
+
+const getToDoList = async function () {
+  return instance
+    .get(`${baseURL}/todos`)
+    .then((res) => res.data)
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
extract this "error handling" mechanism to a constant and reuse it? this later will be replaced with proper monitoring logger